import sys
print(sys.version)
raise Exception("this is a custom exception")
